internal/datastore/memdb/memdb.go

```diff
@@ -100,6 +100,10 @@ type snapshot struct {
 	db       *memdb.MemDB
 }
 
+func (mdb *memdbDatastore) MetricsID() (string, error) {
+	return "memdb-" + mdb.uniqueID, nil
```

Comment:
this almost certainly cause explosion in cadinality for these metrics. E.g. when MemDB SpiceDB is embedded in another process, each restart will bring different uniqueID. I'd personally just have it as `memdb` - the scenario of multiple SpiceDB processes in memory is less likely, and the explosion in cardinality can have severe cost implications

Reply:
Changed